scaleway/helpers_lb.go

```diff
@@ -321,9 +323,11 @@ func flattenLbHCHTTPS(config *lbSDK.HealthCheckHTTPSConfig) interface{} {
 	}
 	return []map[string]interface{}{
 		{
-			"uri":    config.URI,
-			"method": config.Method,
-			"code":   flattenInt32Ptr(config.Code),
+			"uri":         config.URI,
+			"method":      config.Method,
+			"code":        flattenInt32Ptr(config.Code),
+			"host_header": config.HostHeader,
+			"sni":         config.Sni,
```

Comment:
Please verify this on the SDK. Sometimes is a [pointer](https://github.com/scaleway/scaleway-sdk-go/blob/5334351c3723/api/lb/v1/lb_sdk.go#L1658)

Reply:
The sni you are pointing is for the RouteMatch struct, where the one used im my case is for [HealthCheckHTTPSConfig](https://github.com/scaleway/scaleway-sdk-go/blob/master/api/lb/v1/lb_sdk.go#L1288) struct, and its a string